paperoni/webapp/validation.py

```diff
@@ -38,13 +39,26 @@ async def app(page, box):
                     if k.startswith("v-"):
                         paper_id = k.removeprefix("v-")
                         paper = papers[paper_id]
+                        action = None
                         match v:
                             case "valid":
                                 db.insert_flag(paper, "validation", 1)
+                                action = f"{v}ed"
                             case "invalid":
                                 db.insert_flag(paper, "validation", 0)
+                                action = f"{v}ed"
                             case "unknown":
                                 db.remove_flags(paper, "validation")
+                                action = f"{v}ed"
+                        if action is not None:
+                            user = (
+                                page.session.get( "user", {}).get("email", None)
+                            )
+                            db_logger.info(
+                                f"Paper {paper.title} ({paper.paper_id}) "
```

Comment:
It's a binary string, so this will look better:

```suggestion
                                f"Paper {paper.title} ({paper.paper_id.hex()}) "
```

Reply:
fixed